lib/cli.py

```diff
@@ -5,6 +5,7 @@
 import os
 import platform
 import sys
+from math import inf
```

Comment:
Best not to have this import as it's not necessary. Make the default `None `and handle saving if a value is passed